### PR TITLE
Stylin

### PR DESCRIPTION
**Pull request by jooleearr**
DON'T MERGE YET PLEASE! Still need to tidy up the pages other than Games Played. Just opening early so people are aware that the styling set up has changed considerably. Will update the readme before finishing off this week sometime :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,11 +12,17 @@
...
   "devDependencies": {
+    "clean-css": "^3.4.23",
+    "node-sass": "^4.1.1",
+    "nodemon": "^1.11.0",
     "standard": "8.6.0"
   },
   "scripts": {
-    "start": "node app.js",
-    "test": "./node_modules/.bin/standard"
+    "start": "nodemon -e hbs app.js",
```
Comment (Resonance1584) [inline package.json]: Change this to start-dev to not run nodemon in production

**Comment by Resonance1584** [comment]
If it helps - any calls to .style using d3 on https://github.com/endfoos/foosilator/blob/master/views/rankings.hbs can be changed to classes and the styling goes into css

**Comment by Resonance1584** [comment]
@jooleearr any closer to mergeable? - I'm going to start on auth / registration this week.

**Comment by jooleearr** [comment]
@Resonance1584 crap sorry I didn't get to this last week. Will try to get it in a mergeable state tonight after work 👍

**Comment by jooleearr** [comment]
Hey @Resonance1584 - I've whipped through the remaining templates and tidied them up. Still need to sort out heading styles etc properly but looks tidy enough to merge now if you're happy with it. Thanks!

**Pull request merged**